honeybee_vtk/load_config.py

```diff
@@ -0,0 +1,236 @@
+        legend: bool) -> Model:
+    """Get a model with point_in_time data loaded.
+
+    Args:
+        validation: If True, the data will be validated.
+        data: A DataConfig object from the config file.
+        model: A honeybee-vtk model.
+        grid_type: A string indicating whether the model has points and meshes.
+        scene: A honeybee-vtk scene.
+        legend: A boolean to indicate whether the legend should be loaded.
+
+    Returns:
+        A honeybee-vtk model with point_in_time data loaded.
+    """
+    folder_path = pathlib.Path(data.path)
+    identifier = data.identifier
+    object_type = data.object_type
+
+    if validation:
+        _validate_simulation_data(data, model, grid_type)
+    model = _load_point_in_time_data(
+        folder_path, identifier, object_type, model, grid_type)
+    # Legend will only be used in export-images command
+    if legend and data.legend_parameters:
+        _load_legend_parameters(data, model, scene)
+    return model
+
+
+def load_config(json_path: str, model: Model, scene: Scene,
+                validation: bool = False, legend: bool = False,
+                time_series: bool = False, hbjson: str = None) -> Model:
+    """Mount data on model from config json.
+
+    This functions outputs a model or a time series folder based on whether the time
+    series flag is used.
+
+    Args:
+        json_path: File path to the config json file.
+        model: A honeybee-vtk model object.
+        scene: A honeybee-vtk scene object.
+        validation: A boolean indicating whether to validate the data before loading.
+            Defaults to False.
+        legend: A boolean indicating whether to load legend parameters.
+            Defaults to False.
+        time_series: A boolean indicating whether to load time series data.
+            Defaults to False.
+        hbjson: File path to HBJSON. This is only used in the case of time series data.
+            Defaults to None.
+
+    Returns:
+        Either a honeybee-vtk model with data loaded on it or a time series folder.
+    """
+    try:
+        with open(json_path) as fh:
+            config = json.load(fh)
+    except json.decoder.JSONDecodeError:
+        raise TypeError(
+            'Not a valid json file.'
+        )
+    else:
+        result = ''
+
+        for json_obj in config.values():
+            # check if model has grids loaded
+            assert len(model.sensor_grids.data) > 0, 'Sensor grids are not loaded on'\
+                ' this model. Reload them using grid options.'
+
+            data = DataConfig.parse_obj(json_obj)
+            grid_type = _get_grid_type(model)
+
+            if not data.hide:
+                if not time_series:
+                    result = _get_model_with_point_in_time_data(
+                        validation, data, model, grid_type, scene, legend)
+                else:
+                    result = to_time_series_folder(data, model, grid_type, hbjson)
```

Comment:
This is a bit of a code smell. The fact that we need both the Model and the Scene to load the data and config doesn't feel right.

Maybe this method should be called from inside the Scene or Model object? Let's see how it works.